parser/src/java/org/apache/hadoop/hive/ql/parse/HiveParser.g

```diff
@@ -1042,6 +1042,7 @@ ddlStatement
 @init { pushMsg("ddl statement", state); }
 @after { popMsg(state); }
     : createDatabaseStatement
+    | createRemoteDatabaseStatement
```

Comment:
instead of adding a new top level definition called "createRemoteDatabaseStatement", we should add an alternate syntax definition within "createDatabaseStatement". Please refer to createTableStatement syntax.